images/calico/configs/latest.node.apko.yaml

```diff
@@ -1,6 +1,7 @@
 contents:
   packages:
     - calico-node
+    - busybox
```

Comment:
For posterity, why do we need `busybox`?

Reply:
@joshrwolf knows more, but in short this fails without it. I think the pod does that thing where the binary gets copied out so it needs busybox to do the copying

Reply:
the container uses a custom runit script on boot that needs /bin/sh